src/components/timer/Pie.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+
+export default class Pie extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {totalTime: null}
+  }
+
+  componentWillMount() {
+    this.setState({totalTime: this.props.time});
```

Comment:
This is much better. However, since you don't need to mutate the "totalTime" variable anywhere, there is no reason to use setState at all.

refactor this to something like this

```
const Pie = ({time}) => {
  // Put code from render() method here, replacing `this.state.totalTime` with `time`
}

export default Pie
```